src/anchorfm-pupeteer/index.js

```diff
@@ -268,12 +268,6 @@ async function clickSelector(page, selector, options = {}) {
   await clickDom(page, elementHandle);
 }
 
-async function clickXpath(page, xpath, options = {}) {
-  await page.waitForXPath(xpath, options);
-  const [elementHandle] = await page.$x(xpath);
-  await clickDom(page, elementHandle);
-}
```

Comment:
I'm sorry to have not been clear: I was referring to this function being removed.
So people who used the function previously, they won't be able to do it anymore.

Anyway, I'm ok to drop the use of this function

Reply:
Ah yes, you are right, but normally, if someone wants to make a PR, or sync with latest changes from our master, they would need to apply the change in  their code that they maintain for themselves. Good to note in release notes anyway, it would benefit the maintainers of private forks for example.